content/guides/models/tables/visualize-tables.md

```diff
@@ -71,6 +71,71 @@ To view the two tables side-by-side, change the first dropdown from "Merge Table
 * **explore two tables independently**: scroll through and focus on the side/rows of interest
 
 
+
+## Visualize how values change throughout your runs
```

Comment:
Is this actually "during a run"?

Reply:
The idea is to compare different runs. E.g. Suppose you run the same script twice but you changed a single hyperparameter. You would use this feature to compare each step (or whatever unit you chose) to see how the runs compare. E.g. Step 1 of Run A vs Step 1 of Run B.

Reply:
So you are still comparing two runs, that still doesn't seem like "throughout your runs" exactly.